packages/contracts-ts/addresses.json

```diff
@@ -0,0 +1,86 @@
+{
```

Comment:
We are missing tons of contracts that we can find ways to add in follow up pr. Will likely add them with etherscan plugin https://wagmi.sh/cli/plugins/etherscan